server/routes/bookingJourney/selectAdditionalSupportController.ts

```diff
@@ -0,0 +1,49 @@
+import type { RequestHandler } from 'express'
+import { ValidationChain, body, validationResult } from 'express-validator'
+import { BookerService } from '../../services'
+
+export default class AdditionalSupportController {
+  public constructor(private readonly bookerService: BookerService) {}
+
+  public view(): RequestHandler {
+    return async (req, res) => {
+      res.render('pages/bookingJourney/selectAdditionalSupport', {
+        errors: req.flash('errors'),
+        formValues: req.flash('formValues')?.[0] || {},
+        booker: req.session.booker,
+        bookingJourney: req.session.bookingJourney,
+      })
+    }
+  }
+
+  public submit(): RequestHandler {
+    return async (req, res) => {
+      const { bookingJourney } = req.session
+      const errors = validationResult(req)
+
+      if (!errors.isEmpty()) {
+        req.flash('errors', errors.array() as [])
```

Comment:
```suggestion
        req.flash('errors', errors.array())
```
I've set type definitions for `req.flash()` in`server/@types/express/index.d.ts` so *in theory*, shouldn't need to cast to array here (see `selectVisitorsController.ts`